### DISCUSSION TITLE
Can Export/Import users with CLI

### DISCUSSION THREAD

**Discussion by xavier-GitHub76** [apache/superset#28693] [Ideas]
Hello,
with CLI, it's possible to export/import : 
- datasources (superset export-datasources / superset import-datasources)
- dashboards (superset export-dashboards / superset import-dashboards)
- roles (superset fab export-roles / superset fab import-roles)

but  it's not possible to export/import users with CLI

Best regards

**Comment by fbgeobit** [comment]
When in your organization you have tens of dashboards and complex client roles configuration, it turns extremely necessary having an option to export/import user and roles from CLI.

Superset is a good tool, but is not thought for admin use case.
